core/src/main/java/de/danoeh/antennapod/core/storage/DBReader.java

```diff
@@ -10,7 +10,6 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
-import java.util.Date;
```

Comment:
Please revert the change. While the line is really not needed, touching more files than necessary creates unnecessary noise.